blueos_3_ppc64le_ib/compilers.yaml

```diff
@@ -327,10 +327,10 @@ compilers:
 - compiler:
     spec: xl@16.1.1.12
```

Comment:
Because different versions of XL that are identified on the system by date of drop report the same version numbers. If we can use a date as a spec label, that would be preferable, I think.

Reply:
Ah I see. Good to know!